pkgs/data/sgml+xml/schemas/sgml-dtd/docbook/common.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv
+, lib
+, fetchurl
+, unzip
+
+, version
+, src
+}:
+
+let
+  isoents = fetchurl {
+    url = "http://xml.coverpages.org/ISOEnts.zip";
+    sha256 = "1clrkaqnvc1ja4lj8blr0rdlphngkcda3snm7b9jzvcn76d3br6w";
+  };
+in
+
+stdenv.mkDerivation {
+  pname = "docbook-sgml";
+  inherit version;
+
+  nativeBuildInputs = [
+    unzip
+  ];
+
+  dontUnpack = true;
+  dontConfigure = true;
+  dontBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    o=$out/sgml/dtd/docbook-${version}
+    mkdir -p "$o"
+    cd "$o"
+    unzip "${src}"
+    unzip "${isoents}"
+    sed -e "s/iso-/ISO/" -e "s/.gml//" -i docbook.cat
+
+    runHook postInstall
+  '';
+
+  meta = {
+    description = "DTD schemas for SGML variant of DocBook ${version}";
+    maintainers = [ ];
```

Comment:
To be honest, I already spent too much time on this.

The package is providing and ancient and dead version of a standard, no maintenance will occur other than occasional tree-wide initiatives, just like there were no changes since the schemas were introduced in 2011 and 2013 resp.

If someone knows what license this is, they can feel free to fill it in but I could not bother:

> Copyright 1992-2002 HaL Computer Systems, Inc., O'Reilly & Associates, Inc., ArborText, Inc., Fujitsu Software Corporation, Norman Walsh, Sun Microsystems, Inc., and the Organization for the Advancement of Structured Information Standards (OASIS).
> 
> $Id: docbookx.dtd,v 1.13 2002/07/17 15:26:16 nwalsh Exp $
> 
> Permission to use, copy, modify and distribute the DocBook XML DTD and its accompanying documentation for any purpose and without fee is hereby granted in perpetuity, provided that the above copyright notice and this paragraph appear in all copies.  The copyright holders make no representation about the suitability of the DTD for any purpose.  It is provided "as is" without expressed or implied warranty.
> 
> If you modify the DocBook DTD in any way, except for declaring and referencing additional sets of general entities and declaring additional notations, label your DTD as a variant of DocBook.  See the maintenance documentation for more information.

It looks close to either https://spdx.org/licenses/ISC.html or https://spdx.org/licenses/HPND.html but not really.

Reply:
I am sorry that it takes so long, I am trying my best to speed it up, but we can't expect everyone to follow the rules if we don't do it ourselves.

Arch and freebsd have MIT so I would just go with that and be done.

https://github.com/archlinux/svntogit-packages/blob/packages/docbook-sgml/trunk/PKGBUILD
https://cgit.freebsd.org/ports/plain/textproc/docbook-sgml/Makefile

Reply:
The wording is sufficiently different from [MIT](https://spdx.org/licenses/MIT.html) that I do not feel comfortable declaring it as such.